api/envoy/config/trace/v3/datadog.proto

```diff
@@ -16,6 +18,19 @@ option (udpa.annotations.file_status).package_version_status = ACTIVE;
 
 // [#protodoc-title: Datadog tracer]
 
+// Configuration for the Remote Configuration feature.
+message DatadogRemoteConfig {
+  // Allows to configure and change the tracer behaviour from Datadog's user interface.
+  // Default value is ``false``, which disable the feature.
+  // [#comment: This feature can drastically increase the number of connections to the Datadog Agent.
+  // Each tracer regularly polls for configuration updates, and the number of tracers is the product
+  // of the number of listeners and worker threads.]
+  bool enabled = 1;
```

Comment:
I don't think this is needed. The existence of the DatadogRemoteConfig should be enough to turn this on or not.

Reply:
Since the other field is optional, if I remove the `enabled` field. Enabling remote configuration would look like this:

```yaml
typed_config:
"@type": type.googleapis.com/envoy.config.trace.v3.DatadogConfig
  collector_cluster: datadog_agent
  service_name: envoy-demo
  remote_config: {}
```

I personally find it not explicit enought, but what do you think?

Reply:
That's fine we have stuff like this elsewhere. I would just document it.